core/neomutt.c

```diff
@@ -34,25 +34,24 @@
 #include "account.h"
 #include "mailbox.h"
 
-struct NeoMutt *NeoMutt = NULL; ///< Global NeoMutt object
+struct NeoMutt NeoMutt; ///< Global NeoMutt object
 
 /**
  * neomutt_new - Create the main NeoMutt object
  * @param cs Config Set
  * @retval ptr New NeoMutt
  */
-struct NeoMutt *neomutt_new(struct ConfigSet *cs)
+struct NeoMutt neomutt_new(struct ConfigSet *cs)
```

Comment:
s/neomutt_new/neomutt_init/ and take a `struct Neomutt*` as argument

Reply:
applied